notebooks/ImvizExample.ipynb

```diff
@@ -596,35 +596,65 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "d6af9b27-e5b6-45b6-ac85-5d008faa88b1",
+   "id": "0796b7c5",
    "metadata": {},
    "outputs": [],
    "source": [
-    "imviz.save('myimage.png')"
+    "viewer.save('myimage.png')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "96fee8ea-66e3-489c-be27-b06223dd8809",
+   "id": "5e64af2b",
    "metadata": {},
    "source": [
-    "If you want to align the images by WCS instead of pixels, you can run this but it will remove all markers."
+    "If you want to align the images by WCS instead of pixels, you can run this but it will remove all markers added by the default viewer.\n",
+    "\n",
+    "If you have added markers using a tiled viewer you manually created yourself (see below), you have to call the `reset_markers()` method for that viewer first before running the next cell."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "0e59c379-20ec-4125-83eb-07f56ac22f9b",
+   "id": "b4f68d86",
    "metadata": {},
    "outputs": [],
    "source": [
     "imviz.link_data(link_type='wcs')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "7a4f7c3f",
+   "metadata": {},
+   "source": [
+    "If you want a second viewer, click on the \"picture with +\" icon on top. You can also programmatically control the new viewer, as follows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77368ba8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer_2 = imviz.app.get_viewer_by_id('imviz-1')"
+   ]
+  },
+  {
```

Comment:
This tripped me up.  I think probably we should add an `imviz.add_viewer` so that the adding and loading of data into that viewer can all be done in the notebook, but that doesn't have to happen now if it's too much.  Instead the below will just cue the user as to what they need to do.

```suggestion
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Before running the next cell you'll need to use the Data menu to add data to the viewer you just created."
   ]
  },
  {
```
WARNING: I did this by-hand. So I might have mucked up the json. I think it's right but you'll probably want to make sure it works after/if accepting the suggestion

Reply:
I might actually revert the ids changes in the notebook. It is annoying me too much. I'll apply this manually and give you author credit. Hope that's ok!

Reply:
Update: If I can programmatically get the data to show in second viewer, then we don't need this blurb. Stay tuned!

Reply:
I added code to add data to the viewer, so you don't have to manually do it.